### PR TITLE
Add Scrapes table to have a record of each time the scraper is run

### PR DESCRIPTION
**Pull request by kfili**
- add function to scrapers controller to add scrape meta data
  to Scrapes table
- created_on timestamp in scrapes table can be used for date of
  last scrape

### CODE REVIEW

File: app/controllers/scrapers_controller.rb [hunk 1]
```diff
@@ -5,14 +5,23 @@ class ScrapersController < ApplicationController
   # POST /scrapers
   # POST /scrapers.json
   def create
-    HousingListingScraper.run_scrapers().each do |scraper_listing|
+    listing_info, scraper_info = HousingListingScraper.run_scrapers()
```
Comment (mzagaja) [inline app/controllers/scrapers_controller.rb]: FYI Methods without arguments do not requires parenthesis.

File: db/schema.rb [hunk 2]
```diff
@@ -33,4 +33,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "scrapers", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```
Comment (jmartini) [inline db/schema.rb]: I don't think we need this scrapers table, just the scrapes below

File: lib/housing_listing_scraper/padmapper_scraper.rb [hunk 3]
```diff
@@ -22,13 +22,17 @@ def get_listing_URLs_from_search_html(search_html)
...
         latitude = doc.at("meta[name='place:location:latitude']")['content']
         longitude = doc.at("meta[name='place:location:longitude']")['content']
         return ListingInfo.new(description, address, zip_code, latitude, longitude)
-    end 
-end
\ No newline at end of file
+    end
+
+    def get_site_name()
+        return "Paddmaper"
```
Comment (mzagaja) [inline lib/housing_listing_scraper/padmapper_scraper.rb:36]: Would this (along with the search url) make more sense as attributes stored in the database as part of the scrapers table?

**Review by mzagaja — CHANGES_REQUESTED** [review]

**Comment by jmartini** [comment]
I approve of this

**Review by mzagaja — APPROVED** [review]

**Pull request merged**